.env.example

```diff
@@ -0,0 +1,22 @@
+# Logging
+DEBUG=info*,error*,warn*,debug*
+# keys are derived from the mnemonic above
+DEPLOYER_PRIVATE_KEY=0x754fde3f5e60ef2c7649061e06957c29017fe21032a8017132c0078e37f6193a
+SEQUENCER_PRIVATE_KEY=0xd2ab07f7c10ac88d5f86f1b4c1035d5195e81f27dbe62ad65e59cbf88205629b
+TX_INGESTION_SIGNER_KEY=d2ab07f7c10ac88d5f86f1b4c1035d5195e81f27dbe62ad65e59cbf88205629b
+TX_INGESTION_SIGNER_ADDRESS=0x0E0E05Cf14349469ee3B45dc2fcE50E11B9449B8
+MAX_TX_SIZE=90000
+MIN_TX_SIZE=0
+MAX_BATCH_SIZE=50
+POLL_INTERVAL=15000
+NUM_CONFIRMATIONS=0
+RESUBMISSION_TIMEOUT=1000000
+FINALITY_CONFIRMATIONS=0
+RUN_TX_BATCH_SUBMITTER=true
+RUN_STATE_BATCH_SUBMITTER=true
+MAX_BATCH_SUBMISSION_TIME=0
+SAFE_MINIMUM_ETHER_BALANCE=0
+CLEAR_PENDING_TXS=false
+
```

Comment:
I think it would be better if this used `localhost` instead of `l1_chain` and `geth_l2`. Those are dns entries that resolve to the services when running in a docker-compose network